api/app/helpers/application_helper.rb

```diff
@@ -2,4 +2,18 @@
 
 # Base class from which most applications use
 module ApplicationHelper
+	def json(data, include: nil, except: nil)
+		data = JSON.parse(data.to_json.to_s, symbolize_names: true)
```

Comment:
if you just want the keys in data to be symbolized, you can do `data = data.with_indifferent_access` instead of converting it into a json string and then parsing the json.